godel_robot_config/urdf/blending_eff_macro.xacro

```diff
@@ -46,10 +46,10 @@
 			</joint>
 		
 
-			<joint name="${prefix}eff_frame_to_asus_mount" type="fixed">
+			<joint name="${prefix}eff_frame_to_kinect2_mount" type="fixed">
 				<parent link="${prefix}eff_frame"/>
-				<child link="${prefix}asus_mount"/>
-				<origin xyz="0 0 0.150" rpy="3.14 0 2.09"/>
+				<child link="${prefix}kinect2_mount"/>
+				<origin xyz="0 0 0.150" rpy="3.14 0 ${2.09+3.141592653589793}"/>
```

Comment:
You could place the value of PI into a **property** block and use it where is needed.